spec/openapi.yml

```
@@ -286,6 +310,39 @@ paths:
         500:
           $ref: "#/components/responses/500"
 
+  /sandboxes/{sandboxID}/logs:
+    get:
+      description: Get sandbox logs
+      tags: [sandboxes]
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - $ref: "#/components/parameters/sandboxID"
+        - in: query
+          name: start
+          schema:
+            type: integer
+          description: Starting timestamp of the logs that should be returned
```

To match infra specs
```suggestion
          description: Starting timestamp of the logs that should be returned in milliseconds
```